curriculum/challenges/english/25-front-end-development/quiz-css-flexbox/66ed8fe7f45ce3ece4053eb2.md

```diff
@@ -17,439 +17,524 @@ To pass the quiz, you must correctly answer at least 17 of the 20 of the questio
 
 #### --text--
 
-Placeholder question
+What is a flexbox?
 
 #### --distractors--
 
-Placeholder distractor 1
+An two-dimensional model for layout
 
 ---
 
-Placeholder distractor 2
+An three-dimensional model for layout
 
 ---
 
-Placeholder distractor 3
+An multi-dimensional model for layout
 
 #### --answer--
 
-Placeholder answer
+An one-dimensional model for layout 
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What css property is set to enable the flexbox layout for the <div> element?  
 
 #### --distractors--
 
-Placeholder distractor 1
+```css
+
+div { 
+  flex: 1; 
+}
+
```

Comment:
```suggestion
div { 
  flex: 1; 
}
```
No need for additional blank lines here. It applies to other code blocks as well.

Reply:
Ok, I've removed the additional blank lines for all the code blocks